src/Keboola/CsvTable/Table.php

```diff
@@ -22,7 +22,7 @@ class Table extends CsvWriter {
 	/** @var string */
 	protected $name;
 
-	/** @var bool */
+	/** @var bool|null */
 	protected $incremental = null;
```

Comment:
`null` znamena nenastavene, tu sa to pouziva:
https://github.com/keboola/generic-extractor/blob/5f780db38df33395a992e86ea206e5076062433e/src/Configuration/Extractor.php#L189-L192